.github/workflows/build.yml

```diff
@@ -26,8 +33,19 @@ jobs:
         with:
           submodules: true
 
+      - name: Install vcpkg
+        run: |
+          git clone https://github.com/microsoft/vcpkg.git
+          cd vcpkg && ./bootstrap-vcpkg.sh
+          echo "VCPKG_ROOT=${{github.workspace}}/vcpkg" >> $GITHUB_ENV
+          echo "${{github.workspace}}/vcpkg" >> $GITHUB_PATH
+          ./vcpkg integrate install
```

Comment:
I've come to realize that this line is unnecessary as we're using vcpkg in manifest mode. I suspect it can be removed from all our GitHub actions workflows.